Dockerfile

```diff
@@ -3,9 +3,9 @@ FROM node:12-slim
 WORKDIR /usr/src/app
 ENV NODE_ENV=production
 
-COPY package*.json ./
-RUN npm install --only=production
 COPY . ./
+RUN npm install --only=production
```

Comment:
We should also ignore `node_modules`  volume on docker-compose file, in case we have one.

Reply:
good!

Reply:
it's implemented in https://github.com/cantodaruaemergencial/infra/tree/feat/setup-compose-for-application docker-compose.

thanks!